bin/run_unittests.sh

```diff
@@ -98,6 +100,13 @@ function run_cpp_container_tests {
   ./src/container/container_tests
 }
 
+function run_r_container_tests {
+  cd $DIR
+  cd ../containers/R/tests
+  echo "Running R container tests..."
+  ./run_tests.sh
+}
```

Comment:
Since this script is run inside the Docker image built by `ClipperTestsDockerfile`, you need to install R inside that image. That's why the unit tests are failing right now.

Reply:
Try adding these lines at [line 48](https://github.com/ucbrise/clipper/blob/b7ba1df9b64796ba37bd3bb9c717d1c09f5bee6a/dockerfiles/ClipperTestsDockerfile#L47) just below the "Install kubectl section"
```
# Install R
RUN echo "deb http://cran.rstudio.com/bin/linux/ubuntu xenial/" >> /etc/apt/sources.list \
      && gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9 \
      && gpg -a --export E084DAB9 | apt-key add - \
      && apt-get update && apt-get install -y r-base
```